doc-site/components/nve-combobox.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: component
+---
+
+<CodeExamplePreview>
+
+```html
+<div style="height:20vh">
```

Comment:
Ikke bruk style i kodeeksemplene for å gjøre visningen penere. Vi ønsker å ha kodeeksemplene så enkle som mulig og fokusere på det du ønsker å demonstrere.
`<div style="height:20vh">` oppfatter jeg som støy her. 

Du trenger heller ikke ha 10 valg i `values` i alle eksemplene, kun der du trenger så mange for å vise hvordan komponenten oppfører seg hvis den ikke har plass til å vise alle valgene, enten i popup eller i hva som er valgt.

Vet at dette med enkle eksempler har vært dårlig kommunisert, så jeg har rettet litt i dokumentasjonen nå (se #536).